crates/ruff/src/rules/pydocstyle/mod.rs

```diff
@@ -68,6 +68,7 @@ mod tests {
     #[test_case(Rule::UndocumentedPublicPackage, Path::new("D104/__init__.py"); "D104_1")]
     #[test_case(Rule::SectionNameEndsInColon, Path::new("D.py"); "D416")]
     #[test_case(Rule::SectionNotOverIndented, Path::new("sections.py"); "D214")]
+    #[test_case(Rule::SectionNotOverIndented, Path::new("D214_module.py"); "D214_module")]
```

Comment:
What's up with the naming nomenclature for these files/test? It's kind of chaotic and I did not see clear guidelines in the contributing docs.

Reply:
It's mostly path dependence. We started with _just_ `D.py`, then added `sections.py` at some point when we needed some section-specific tests, and then we added a few cases that are specific to individual rules (like `D104/__init__.py`, which requires multiple files to test). Maybe we just rename this file to `module.py` to indicate that it's a test for module-level docstring behavior?

Reply:
🤷‍♀️ I don't have a strong preference. I named it this because it only contains D214 violations — maybe if we add more module-level docstring errors in it in the future it'd be renamed?

Does the string argument after the path need to be unique?

Reply:
Yeah, IIRC the path has to be unique.